revealer_bot/launch_bot.py

```diff
@@ -1,24 +1,17 @@
 import os
 
 from bot_lair import the_actual_revealer_bot
-from message_parser import parse_message
+
 from revealer_bot.bob_and_other_networky_things import bob
 
 bot_token = os.environ["DISCORD_BOT_TOKEN"]
 
 
 @the_actual_revealer_bot.event
 async def on_ready():
-    print(f'Logged in as {the_actual_revealer_bot.user.name} ({the_actual_revealer_bot.user.id})')
-    print('------synced------')
-
+    print(f"Logged in as {the_actual_revealer_bot.user.name} ({the_actual_revealer_bot.user.id})")
+    print("------synced------")
 
-from revealer_commands import *
```

Comment:
Oh, just realized  - removing this import might break the bot, because the commands registration is hiddeng in the `from revealer_commands import *`, which changes the global bot instance `the_actual_revealer_bot` by using the `@the_actual_revealer_bot.tree.command()` decorator?

Reply:
If that is the case - that is a nasty import side-effect